rskj-core/src/test/java/co/rsk/peg/BridgeSupportRegisterBtcTransactionTest.java

```diff
@@ -2487,7 +2488,8 @@ void flyover_segwit_as_migration_utxo_with_many_outputs_and_inputs(
         FederationTestUtils.addSignatures(retiringFederation, retiringFedSigners, btcTransaction);
 
         TransactionWitness txWitness = new TransactionWitness(1);
-        txWitness.setPush(0, new byte[]{0x1});
+        txWitness.setPush(0, new byte[72]); // push for signatures
+        txWitness.setPush(1, retiringFederation.getRedeemScript().getProgram());
```

Comment:
Should we have a test util that creates a witness for a given redeem script? Perhaps with/without signatures?